install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2276,6 +2276,13 @@ routingMode: ""
 # @default -- Port 8472 for VXLAN, Port 6081 for Geneve
 tunnelPort: 0
 
+# -- Configure what the response should be to traffic for a service without backends.
+# "reject" only works on kernels >= 5.10, on lower kernels we fallback to "drop".
+# Possible values:
+#  - reject (default)
```

Comment:
@dylandreimerink is the >= 5.10 requirement correct? We later say that the required BPF feature (`BPF_ADJ_ROOM_MAC` support in `bpf_skb_adjust_room()`) exists since kernel 5.2.

Reply:
Yes, I believe you are correct. That should be 5.2, I am not sure how 5.10 ended up there.

Reply:
ack, I glanced at a 5.4 sysdump and it looks all golden there. No error message from the probe.

I'll open a PR to just remove this on v1.16+, given all supported kernels should be providing the option.

Reply:
https://github.com/cilium/cilium/pull/35734